scripts/planktoscope/segmenter/__init__.py

```diff
@@ -813,11 +813,16 @@ def segment_path(self, path, ecotaxa_export):
             )
         )
 
-        project = self.__global_metadata["sample_project"].replace(" ", "_")
-        sample = self.__global_metadata["sample_id"].replace(" ", "_")
-        date = datetime.datetime.utcnow().isoformat()
+        def clean_name(name):
```

Comment:
This should be extracted to its own utils.py module.
We are going to need this function in the imager too: https://github.com/PlanktonPlanet/PlanktoScope/blob/7ef2cd7eabca4b9352e61f4dd131edb2ec8439c8/scripts/planktoscope/imager/__init__.py#L563

Then you just have to `import planktoscope.utils` and call the function. It's a bit more verbose, but more flexibility down the way.

Reply:
good idea!
Haven't looked at that file yet